rpc-rs/src/provider_main.rs

```diff
@@ -68,14 +68,11 @@ static BRIDGE: OnceCell<HostBridge> = OnceCell::new();
 
 // this may be called any time after initialization
 pub fn get_host_bridge() -> &'static HostBridge {
-    match BRIDGE.get() {
-        Some(b) => b,
-        None => {
-            // initialized first thing, so this shouldn't happen
-            eprintln!("BRIDGE not initialized");
-            panic!();
-        }
-    }
+    BRIDGE.get().unwrap_or_else(|| {
+        // initialized first thing, so this shouldn't happen
+        eprintln!("BRIDGE not initialized");
+        panic!();
```

Comment:
it would happen right when an exe process starts, and it tells you there was a fundamental mis-use of the api. It's not the kind of bug that would happen spuriously in a server.